extensions/adobe/experience/campaign/orchestration/experienceevent.schema.json

```diff
@@ -0,0 +1,118 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":
+    "https://ns.adobe.com/experience/campaign/orchestration/experienceevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "User journey experience event",
+  "type": "object",
+  "description": "Orchestration extension to ExperienceEvent",
+  "definitions": {
+    "journey": {
+      "title": "Journey for users",
+      "type": "object",
+      "description": "Journey created by marketer for its users",
+      "properties": {
+        "@id": {
+          "title": "Journey unique identifier",
+          "type": "string",
+          "format": "uri",
+          "description": "The unique identifier of the journey created by the marketer."
+        }
+      },
+      "required": ["@id"]
+    },
+    "journeyVersion": {
+      "title": "Journey version",
+      "type": "object",
```

Comment:
Why do you need this complex definition for a version?  And where is the specific numbering of the version - all I see is a unique ID

Reply:
At this moment, I just pulled out the required fields reporting event. In near future, definitely we will add all attributes to journeyVersion. Those changes were also not finalized yet, thats why I kept "object" for future proof.